content/api/commands/session.md

```diff
@@ -0,0 +1,731 @@
+---
+title: session
+---
+
+Cache and restore [cookies](/api/cypress-api/cookies),
+[`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage),
+and
+[`sessionStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage)
+in order to reduce test setup times.
```

Comment:
One point of improvement would be to think about how to better talk about the value of the API beyond stating "test setup times." I anticipate that this doesn't really convey the immediate value and could be misinterpreted. 

I have no actionable edit at the moment, but it's important to think more about this.

Reply:
If there's anything from the [(draft) blog post](https://cypress-io.ghost.io/blog/p/32959945-4e57-40d7-8a47-8e1ba67784cf/) that you think will add value here, let me known. I just want to make sure that we don't make this page more difficult to consume as API documentation!

Reply:
I'll take another look at the post. Sidenote, we should also link to he blog post in the "See Also" section.

Reply:
Link added in ecefd272a47a77cdb1391356c3f9ee3d344ec6b9